### ISSUE TITLE
Authentication: Public/Private key (for users)

### ISSUE THREAD

**Issue by Harmageddon** [openbase/bco.authentication#49]
Public/private key authentication is only implemented for devices so far.

**Comment by DivineThreepwood** [comment]
@pLeminoq this has already been implemented, right?
This is how the eclipse.smarthome.binding is currently authenticated, right?

If this issue can be closed, please close the related epic as well.

**Comment by pLeminoq** [comment]
No, this is not implemented. The binding and the openhab device manager both login as a device (or a client in authentication terms) to be able to use public-private-key authentication. This issue is blocked by #72 because without it the authenticator cannot decide if a password hash or a public key is stored in its credential store and thus does not know if symmetric or asymmetric encryption needs to be used.

**Comment by DivineThreepwood** [comment]
alright, issue will be moved back to icebox.
Seems to be a non api changing feature so its target version will be 2.1
